devimint/src/external.rs

```diff
@@ -661,6 +668,131 @@ pub async fn open_channel(
     Ok(())
 }
 
+// TODO(tvolk131): Remove the `cln` and `lnd` arguments and instead use `gw_cln`
+// and `gw_lnd`
+pub async fn open_channel_between_gateways(
+    process_mgr: &ProcessManager,
+    bitcoind: &Bitcoind,
+    cln: &Lightningd,
+    gw_cln: &Gatewayd,
+    lnd: &Lnd,
+    gw_lnd: &Gatewayd,
```

Comment:
Not a blocker right now but we'll need to think about how to restructure this for LDK

Reply:
Yep! My goal is to remove the `cln` and `lnd` arguments, then rename `gw_cln` and `gw_lnd` to something more generic, like `gw1` and `gw2`. Then an LDK gateway will just be another instance of `Gatewayd`.